### DISCUSSION TITLE
Question: Online

### DISCUSSION THREAD

**Discussion by TNT-King** [HaxeFlixel/flixel#2451] [Q&A] [answered]
Hi.
Soo...
...
...I am very noob at coding
...
Soo i was just wondering
...
How can i make a multiplayer online game???

**Comment by TheHENOOB** [comment, marked as answer]
https://colyseus.io (requires knowledge on servers and node.js (javascript) )

I don't recommend trying to make a multiplayer right now if you are "noob at coding", you should be looking for tutorials, documentation and demos to learn more.

**Reply by TNT-King** [reply]
Thanks. I'll try :D

**Reply by Geokureli** [reply]
Here's a video tutorial for Colyseus with HaxeFlixel
https://www.youtube.com/watch?v=5XaGzRH5Y7I

**Comment by DigiEggz** [comment]
Once you become comfortable with coding, you can also check out SmartFoxServer for online multiplayer.

I help maintain a Haxe version that can be found here: https://github.com/DigiEggz/smartfox-haxe-client

Good luck in your journey!
